test/image_test.py

```diff
@@ -369,8 +368,8 @@ def test_save_colorkey(self):
     def test_load_unicode_path(self):
         import shutil
 
-        orig = unicode_(example_path("data/asprite.bmp"))
-        temp = os.path.join(unicode_(example_path("data")), u"你好.bmp")
+        orig = str(example_path("data/asprite.bmp"))
+        temp = os.path.join(str(example_path("data")), u"你好.bmp")
```

Comment:
example_path probably returns string all the time now?

Reply:
Fixed in 768fda1 (and same fix applied in many lines in many files).